endpointslices/endpointslices.go

```diff
@@ -94,8 +92,11 @@ func createEPSliceInfos(epSlicesList *discoveryv1.EndpointSliceList, servicesLis
 		ownerRef := epSlice.OwnerReferences[0]
 		name := ownerRef.Name
 		namespace := epSlice.Namespace
-		if service, found = getService(name, namespace, servicesList); !found {
+		var svc corev1.Service
+		if service := getService(name, namespace, servicesList); service == nil {
 			return nil, fmt.Errorf("failed to get service for endpoint %s/%s", epSlice.Namespace, epSlice.Name)
+		} else {
+			svc = *service
```

Comment:
nit: just deref service at the bottom, the "svc" var just confuses

Reply:
fixed, thanks. I don't know why it didn't work for me when I first tried it.